app/commands/v2/unpublish.rb

```diff
@@ -132,7 +132,9 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
```

Comment:
From my limited understanding of the state in the Publishing API, there should only be a maximum of 1 unpublished/published content item at any one time.

I really need to learn some more about this, as its unclear to me, which is very bad...